canonical-web-and-design/ubuntu.com
# Updates /openstack/install to reference /openstack/tutorials

## Done

- Updates /openstack/install to reference /openstack/tutorials as per [copydoc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against copydoc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4556
